assets/shared.css

```diff
@@ -1,10 +1,13 @@
 :root {
     --bg-color: #1e191e;
+    --bg-color-secondary: #38343e;
```

Comment:
If this becomes unused because of the removal of its only usage, it should obviously be deleted from here too.